book/src/development/common_tools_writing_lints.md

```diff
@@ -265,10 +265,10 @@ functions to deal with macros:
    ```
 
 [Ty]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/ty/struct.Ty.html
-[TyKind]: https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/ty/enum.TyKind.html
+[TyKind]: https://doc.rust-lang.org/nightly/nightly-rustc/stable_mir/ty/enum.TyKind.html
```

Comment:
I don't think stable_mir is the right one? Shouldn't it be https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/ty/type.TyKind.html ?

Reply:
I would say this one https://doc.rust-lang.org/nightly/nightly-rustc/rustc_type_ir/ty_kind/enum.TyKind.html